conda/core/repodata.py

```diff
@@ -420,6 +420,12 @@ def fetch_repodata(url, schannel, priority,
     else:
         mod_etag_headers = read_mod_and_etag(cache_path)
 
+        if use_cache:
+            log.debug("Using cached repodata for %s at %s because use_cache=True",
+                      url, cache_path)
+            return read_local_repodata(cache_path, url, schannel, priority,
+                                       mod_etag_headers.get('_etag'), mod_etag_headers.get('_mod'))
+
```

Comment:
I was about to point out a problem that `cache_path` might not actually exist.  But it looks like we've actually take care of that situation in line 416 above.  I think we can go ahead with this PR.  As it currently exists, I think #5146 might be problematic.